gmso/external/convert_dataframe.py

```diff
@@ -270,8 +273,19 @@ def _parse_unyts_to_headers(dataList, columns) -> list:
 def _generate_component_lists(topology, parameter, columns) -> list:
     outList = []
     columnsList = []
+    #Customize warning message
+    def custom_formatwarning(msg, *args, **kwargs):
```

Comment:
Let's delete this warning formatter. I think it's cool, but looks like we're not doing any custom formatting, since the "\n" I think is already part of the default behavior.